cloudconfig/containerinit/container_userdata.go

```diff
@@ -202,17 +202,19 @@ func PrepareNetworkConfigFromInterfaces(interfaces []network.InterfaceInfo) *Pre
 // might include per-interface networking config if both networkConfig
 // is not nil and its Interfaces field is not empty.
```

Comment:
To my reading of this comment, this change makes it more correct, in that we can now have networkConfig that is nil.